src/features/components/atoms/tab/tab-content/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from 'react'
+import { TStepperTab, TTabItem } from '../types'
+import FlexBox from 'features/components/atoms/flex-box'
+import Box from 'features/components/atoms/box'
+import Typography from 'features/components/atoms/typography'
+import useBreakpoints from 'components/hooks/use-breakpoints'
+import { Localize } from 'components/localization'
+
+const TabContent = ({ items: initialItems, tab, direction = 'row' }: TStepperTab) => {
+    const [selected, setSelected] = useState<number>(0)
+    const [items, setItems] = useState<TTabItem[]>(initialItems)
+    const { is_mobile_or_tablet } = useBreakpoints()
+
+    const image_style = !is_mobile_or_tablet ? { blockSize: '62.5rem' } : { blockSize: '43.5rem' }
+
+    useEffect(() => {
+        setItems(initialItems)
+    }, [tab, initialItems])
```

Comment:
since we are not doing anything with tab 
do we need tab as a dependency?

Reply:
We change a tab both with initial_items there
Found a mistake with camelCase...